mergedSchema.js

```diff
@@ -0,0 +1,46 @@
+import { mergeSchemas, introspectSchema, makeRemoteExecutableSchema } from "graphql-tools"
+import { createHttpLink } from "apollo-link-http"
+import fetch from "node-fetch"
+
+import localSchema from "./schema"
+
+export default async function mergedSchema() {
+  const convectionLink = createHttpLink({
+    fetch,
+    uri: process.env.CONVECTION_GRAPH_URL,
+    headers: {
+      Authorization: `Bearer ${process.env.CONVECTION_TOKEN}`,
```

Comment:
@sweir27 I guess this actually needs to be specified by the client at runtime, as it differs per user, right? I think that should be easy using [the ‘link’ API](https://www.apollographql.com/docs/graphql-tools/remote-schemas.html#link-api).

Reply:
Yeah, [the convection loaders](https://github.com/artsy/metaphysics/blob/master/lib/loaders/loaders_with_authentication/convection.js) have the ability to convert a grav token into a convection token. We'll need something like this for at least convection / impulse

Reply:
Ooohhh, we can probably just access those data loaders (including the token loader) during stitching resolving. Will take a look 👍